.github/workflows/tests.yml

```diff
@@ -7,51 +7,40 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
+    branches: [ master, v2/dev ]
 
 jobs:
   build:
 
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]  # TODO: fix windows permissions issues and add windows-latest
-        python-version: [3.7, 3.8]
-        exclude:
-        # excludes node 8 on macOS
-          - os: macos-latest
-            python-version: 3.8
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.11]
 
     steps:
     - uses: actions/checkout@v2
     - uses: conda-incubator/setup-miniconda@v2
       with:
-        python-version: ${{ matrix.python-version }}
```

Comment:
Is the reason for removing this line just that we're only testing on version 3.11 now? Does it hurt to leave it and also test on 3.12?

Reply:
This was removed because at the time 3.12 was not out, adding it back now (and python 3.12)